packages/terra-collapsible-menu-view/src/CollapsibleMenuViewDivider.jsx

```diff
@@ -2,22 +2,45 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Menu from 'terra-menu';
+import IconChevronRight from 'terra-icon/lib/icon/IconChevronRight';
 import ThemeContext from 'terra-theme-context';
 import styles from './CollapsibleMenuView.module.scss';
 
 const cx = classNames.bind(styles);
 
+const propTypes = {
+  /**
+   * @private
+   * Uses a ">" icon instead of a line.
+   */
+  useChevronRightIcon: PropTypes.bool,
+};
+
 const contextTypes = {
   isCollapsibleMenuItem: PropTypes.bool,
 };
 
-const CollapsibleMenuViewDivider = (props, { isCollapsibleMenuItem }) => {
+const CollapsibleMenuViewDivider = ({ useChevronRightIcon }, { isCollapsibleMenuItem }) => {
   const theme = React.useContext(ThemeContext);
+  if (useChevronRightIcon) {
+    if (isCollapsibleMenuItem) {
+      return null;
+    }
+
+    return (
+      <div className={cx(['chevron-right', 'face-up-item', theme.className])}>
```

Comment:
Why do we need an additional div element around the icon?

Reply:
I had a css issue where the svg of the icon was getting squished by the padding size. This div ensures that the svg will always be at its appropriate size

Reply:
Did you try using margin instead of padding?

Reply:
Margin works however with the way collapsible menu view is setup, it calls getBoundingClientRect on each child, which ignores margins. So it was throwing off resizing calculations